.github/workflows/pull-request.yml

```
@@ -312,22 +332,4 @@
-  #       uses: actions/checkout@v4.2.2
-  #     - name: Create .env file for talawa api testing environment
-  #       run: cp ./envFiles/.env.ci ./.env
-  #     - name: Build talawa api non production environment docker image
-  #       run: docker compose build
-  #     - name: Run import:sample-data
-  #       uses: addnab/docker-run-action@v3
-  #       with:
-  #         image: talawa_api
-  #         options: --env-file ./envFiles/.env.ci
-  #         run: pnpm run import:sample-data
+          python .github/workflows/scripts/check_docstrings.py --directories .github
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)